src/transformers/models/wav2vec2_conformer/convert_wav2vec2_conformer_original_pytorch_checkpoint_to_pytorch.py

```diff
@@ -37,6 +39,7 @@
 logging.set_verbosity_info()
 logger = logging.get_logger(__name__)
 
+SAMPLE_RATE = 8000  # 8000, 16000
```

Comment:
Rather than hard-coding the `SAMPLE_RATE`, which is going to break the existing conversions, can we instead leave it as an argument that the user can pass to the script? (in the same way we pass the `checkpoint_path` etc)

It would also be great to have this `sample_rate` argument default to 16kHz, rather than 8kHz, for backward compatibility.